gravitybee/__init__.py

```diff
@@ -457,15 +466,16 @@ def _create_hook(self):
             {'app_name': self.args.info["app_name"]})
 
         # 1 - extra data
-        hook += "# collection extra data, if any (using --extra-data option)"
-        for data in self.args.extra.get("data", []):
-            hook += "\ndatas.append(('"
-            hook += self.args.directories["pkg"] + os.sep
-            if self.args.directories["src"] != '.':
-                hook += self.args.directories["src"] + os.sep
-            hook += self.args.info["pkg_name"] + os.sep + data
-            hook += "', '" + self.args.info["pkg_name"] + "/" + data + "'))"
-            hook += "\n\n"
+        if self.args.extra["data"] is not None:
+            hook += "# collection extra data, if any (using --extra-data option)"
+            for data in self.args.extra.get("data", []):
```

Comment:
Interesting... So you have a package where `data` is set, but the value is `None`? In that case, instead of the `if` and the extra nesting, can we just change this loop like so:

```suggestion
            for data in self.args.extra.get("data") or []:
```

Reply:
@nikolarobottesla I went ahead and updated the branch with what I meant. Can you confirm the branch is working for you with conda and the package you are building?